lib/WorkflowRnaseq.groovy

```diff
@@ -65,13 +65,10 @@ class WorkflowRnaseq {
                 Nextflow.error("Invalid option: '${params.aligner}'. Valid options for '--aligner': ${valid_params['aligners'].join(', ')}.")
             }
         } else {
-            if (!params.pseudo_aligner) {
-                Nextflow.error("--skip_alignment specified without --pseudo_aligner...please specify e.g. --pseudo_aligner ${valid_params['pseudoaligners'][0]}.")
-            }
             skipAlignmentWarn(log)
         }
 
-        if (params.pseudo_aligner) {
+        if (!params.skip_pseudo_alignment) {
```

Comment:
Also, this logic seems to have been mangled?

Previously:
if `params.pseudoaligner` has been specified, check that a valid one has been chosen from the hardcoded `valid_params`. 

Now:
if `skip_pseudo_alignment` has not been specified, check if `valid_params['pseudoaligners']` contains `params['pseudoaligner']` (`null` for the default alignment-based routes?) and otherwise raise a Nextflow error?

Reply:
Done https://github.com/nf-core/rnaseq/pull/1039/commits/993fff67ba7590c85cb107143513943ff39dbd21